### DISCUSSION TITLE
Timelapse Video Download Issue

### DISCUSSION THREAD

**Discussion by Mkel2010** [OrcaSlicer/OrcaSlicer#4254] [General]
I'm having issues with downloading Timelapse videos from the Device tab in Orca. I can see the videos and play them, but the download option for a rendered video does not work in the Mac version. 

If I go to the Timelapse section of Fluidd and left click a rendered video, I get a dialog box that retrieves the video and opens it in the same window. At this point I still have all the tabs down the left side that would allow me to switch back to the Home Screen or other options; clicking on the Home icon takes me back to the Timelapse file list at this point and I could then go back to the Fluidd Home Screen if necessary. I can play the video at this point.  As soon as I select the Download button at the bottom of this screen, the video seems to be opened in QuickTime Player within Orca Slicer and all the icons on the left disappear. I am unable to switch back to the Fluidd Home Screen or do anything else within the Device tab. Right clicking on the video does give an option to download the video but nothing happens. At this point the only way to get the Device Tab functionality back is to quit Orca Slicer and re-open it. This has to be done from the Task Bar menu as Command-Q ceases to function at this point.

I can open Fluidd in a regular web browser and download the videos using the Download button described above. This seems to be a bug with the Mac version (I am using the current stable release of Orca.)

**Comment by brentcklein** [comment]
I'm also still having this same issue on Mac. Version 2.3.0, build 70931e5.
